### DISCUSSION TITLE
How to use ExecStartPre in Quadlet units?

### DISCUSSION THREAD

**Discussion by dimon222** [containers/podman#20642] [Q&A] [answered]
Basically title. I currently use modified podman systemd generated units and was considering to switch to Quadlet units, but can't find the way to specify my custom ExecStartPre in Quadlet. Is there a way?

**Comment by dimon222** [comment, marked as answer]
I'm dumb, it's undocumented but is accessible out of the box.
https://github.com/containers/podman/discussions/19834

**Reply by rhatdan** [reply]
Basically you can put any systemd content from the service section into a quadlet, would it have helped if we stated that? Open a PR on how you think the man page should change to help future people understanding it.

**Reply by dimon222** [reply]
Yes, I believe it would be great if it was mentioned + add some setting to example unit.
